x-pack/plugins/monitoring/kibana.json

```diff
@@ -24,5 +24,5 @@
   ],
   "server": true,
   "ui": true,
-  "requiredBundles": ["kibanaUtils", "home", "alerts", "kibanaReact", "licenseManagement", "triggersActionsUi"]
+  "requiredBundles": ["kibanaUtils", "home", "alerts", "kibanaReact", "licenseManagement"]
```

Comment:
Why are we able to not add this as a required bundle? Maybe I don't understand the change enough with this PR, but is it true that the changes enable us to remove this?

Reply:
We currently exposed flyout in the different way by the triggers_actions_ui startup contract, and because of that the bundle of the plugin is not required any more.
And I've got the next error on Kibana start :-) 
```
ERROR in ../kibana/x-pack/plugins/monitoring/kibana.json
       │          Bundle for [monitoring] lists [triggersActionsUi] as a required bundle, but does not use it. Please remove it.
```